src/constants.ts

```diff
@@ -33,11 +33,13 @@ export const COMMIT_FILE_NAMES = {
   LICENSE: 'LICENSE',
 };
 
+const COMMIT_SIGN_OFF = '\nSigned-off-by: repomountie pathfinder@gov.bc.ca'
```

Comment:
approving this but it is better to leave string formatters outside of a message like this.

Reply:
;

Reply:
I think it needs to be 
```suggestion
const COMMIT_SIGN_OFF = `Signed-off-by: ${BOT_NAME} <pathfinder@gov.bc.ca>`
```